sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -79,11 +79,9 @@ public class OpenTelemetryTracer implements com.azure.core.util.tracing.Tracer {
             // to resolve the actual io.opentelemetry.context.Context.
             applicationContextClass = Class.forName("#io.opentelemetry.context.Context".substring(1));
             convertAppToAgentContextMethod = agentCtxStorageClass.getMethod("getAgentContext", applicationContextClass);
-        } catch (ReflectiveOperationException t) {
+        } catch (ReflectiveOperationException | RuntimeException t) {
```

Comment:
LinkageError doesn't extend RuntimeException, plus maybe good to limit scope just to LinkageError?

```suggestion
        } catch (ReflectiveOperationException | LinkageError t) {
```

Reply:
some related examples: https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-java-instrumentation%20LinkageError&type=code

Reply:
yeah, I thought about it and decided that really it does not matter what happened, either way we should continue. 
If context conversion throws later, we'd back off and never try again. If it won't work - that's not the end of the world either.

So there is no point in throwing - it won't help anything.

Reply:
> So there is no point in throwing - it won't help anything.

makes sense

I wasn't sure if you also wanted to catch `LinkageError` here (currently it doesn't since `LinkageError` doesn't extend `RuntimeException`)